Makefile.am

```diff
@@ -221,14 +221,16 @@ dist_check_SCRIPTS += test/functional/generate-cert.prereq
 
 parallel_tests = $(BATS:.bats=.log)
 $(parallel_tests) : test/functional/generate-cert.log
-endif
 
+install-check:
+	test/installation/test.bats
 
+endif
 
 if COVERAGE
 AM_CFLAGS += --coverage
 
-coverage: coverage-clean
+coverage: coverage-clean ]
```

Comment:
This looks like an accidental change

Reply:
I wonder how this is not breaking any test. I'll remove that. it was an accident. thanks